### ISSUE TITLE
Build Problem with Ubuntu using --prefix while configuring PostgreSQL

### ISSUE THREAD

**Issue by aamederen** [citusdata/citus#384]
Citus build (make) fails if PostgreSQL is configured with --prefix=/some/custom/pg/dir command. 

Following list of commands successfully compiles Citus: 
```
cd /home/ubuntu/postgresql-9.5.1
make clean
./configure
make -j 8
sudo make install
export PATH=/usr/local/pgsql/bin:$PATH
cd /home/ubuntu/citus
make clean
./configure
make
sudo make install
```

Follow these commands (on a different terminal) to reproduce the error:
```
export PATH=/home/ubuntu/pgdev/bin:$PATH
cd /home/ubuntu/postgresql-9.5.1
make clean
./configure --prefix=/home/ubuntu/pgdev
make -j 8
sudo make install
cd /home/ubuntu/citus
make clean
./configure
make
```

The error message is this:
```
...
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -O2 -fpic -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers -Wno-clobbered -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wmissing-declarations -Wmissing-prototypes -I '/home/ubuntu/citus/src/include' -I. -I./ -I/home/ubuntu/pgdev/include/postgresql/server -I/home/ubuntu/pgdev/include/postgresql/internal -D_GNU_SOURCE  -I/home/ubuntu/pgdev/include  -c -o test/connection_utils.o test/connection_utils.c
test/connection_utils.c:16:24: fatal error: internal/c.h: No such file or directory
 #include "internal/c.h"
                        ^
compilation terminated.
make[1]: *** [test/connection_utils.o] Error 1
make[1]: Leaving directory `/home/ubuntu/citus/src/backend/distributed'
make: *** [extension] Error 2

```

Tried this on a fresh Ubuntu 14.04 Amazon EC2 m3.large instance with PostgreSQL 9.5.1 and latest Citus:
```
$ uname -a
Linux ip-10-180-95-46 3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

Issue occurs with [6925639] (https://github.com/citusdata/citus/commit/692563930fcea656df883b2034220315ce86719f) and [a53fb90] (https://github.com/citusdata/citus/commit/a53fb90ef94ff3d2097539eeee38cc0b5bdf8d55) but does not occur with [297bd57] (https://github.com/citusdata/citus/commit/297bd5768d818c8fbe3c07ca397b1f3166f582eb).

@jasonmp85, do you have any idea?

**Comment by anarazel** [comment]
Could you recheck that this is still present?

**Comment by begriffs** [comment]
I just got the same error on OS X.

```bash
cd /tmp
mkdir citus-tutorial
wget https://ftp.postgresql.org/pub/source/v9.5.3/postgresql-9.5.3.tar.bz2
bunzip2 postgresql-9.5.3.tar.bz2
tar xf postgresql-9.5.3.tar
cd postgresql-9.5.3
./configure --prefix=/tmp/citus-tutorial --disable-rpath
make
make install

export PATH=/tmp/citus-tutorial/bin:$PATH
cd /tmp
git clone https://github.com/citusdata/citus.git
cd citus
git checkout v5.0.1
./configure
make

##
## make output ...
##

gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -Wno-unused-command-line-argument -O2  -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wmissing-declarations -Wmissing-prototypes -I '/tmp/citus/src/include' -I. -I./ -I/private/tmp/citus-tutorial/include/postgresql/server -I/private/tmp/citus-tutorial/include/postgresql/internal  -I/private/tmp/citus-tutorial/include  -c -o test/connection_utils.o test/connection_utils.c
test/connection_utils.c:16:10: fatal error: 'internal/c.h' file not found
#include "internal/c.h"
         ^
1 error generated.
make[1]: *** [test/connection_utils.o] Error 1
make: *** [extension] Error 2
```

**Comment by anarazel** [comment]
But it's fixed in master, right?
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by begriffs** [comment]
Yep just checked and it is fixed on master.

**Comment by jasonmp85** [comment]
Closing this because this is fixed in master, 5.1-rc.1, etc.
